amenity-points.mss

```diff
@@ -674,7 +673,18 @@
       marker-opacity: 0.33;
     }
   }
-
+  
+  [feature = 'amenity_recycling'][recycling_type = 'container'][zoom >= 19],
+  [feature = 'amenity_recycling'][zoom >= 19] {
```

Comment:
It looks quite readable, but it means that `[feature = 'amenity_recycling'][zoom >= 19]` covers also recycling centers on z19+. It might accidentally work because of the order of rules Carto CSS is applying, but it would be harder to debug problems. I think instead of these two lines we should just use something like `[feature = 'amenity_recycling'][recycling_type != 'centre'][zoom >= 19]`.